packages/frontend/core/src/components/app-sidebar/sidebar-header/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { useAtomValue } from 'jotai';
+
+import { NavigationButtons } from '../../../modules/navigation';
+import { navHeaderStyle } from '../index.css';
+import { appSidebarOpenAtom } from '../index.jotai';
+import { SidebarSwitch } from './sidebar-switch';
+
+export const SidebarHeader = () => {
+  const open = useAtomValue(appSidebarOpenAtom);
+
+  return (
+    <div
+      className={navHeaderStyle}
+      data-open={open}
+      data-is-macos-electron={environment.isDesktop && environment.isMacOs}
```

Comment:
which place use this?

Reply:
Used in css, leaving space for traffic lights on macos.